### PR TITLE
connects to 127 - fix reloading issue

### PR DESCRIPTION
**Pull request by mugwump303**
This PR closes #[127]
connects to #127

#### What does this PR do?
Uses new user data to update components without reloading page.

### CODE REVIEW

File: server/.eslintrc [hunk 1]
```diff
@@ -8,4 +8,4 @@
   "rules": {
     "no-console": "off"
   }
-}
+}
```
Comment (wdoug) [inline server/.eslintrc:11]: Why're we removing the newline at the end of this file?

Comment (mugwump303) [reply server/.eslintrc:11]: Not sure. I didn't even think about it when I pasted it back in. I can put it back in.

**Review by wdoug — CHANGES_REQUESTED** [review]
I'm not crazy about this solution. The underlying Auth system that we have right now isn't set up to work properly with React which is forcing us to use hacks like `forceUpdate`. I'm going to try to refactor the Auth stuff to work better with React, which should allow us to manage this better and set us up better for the future. Hopefully that will go smoothly. If not, I will approve this and we can merge it.

**Comment by wdoug** [comment]
This PR is superseded by #178

**Pull request closed without merging**